modules/example/src/main/scala/example/api/ErrorResponse.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 LambdaWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example.api
+
+import zio.Chunk
+import zio.json.{DeriveJsonEncoder, JsonEncoder}
+
+final case class ErrorResponse(errors: ErrorResponseData)
```

Comment:
Does this structure result in a structure like:
```
{
   errors: {
      body: [
         "This is an error"
      ]
   }
}
```
?
Can we simplify that and also avoid naming a class with `Data`?